README.md

```diff
@@ -1 +1 @@
-# test-cred
\ No newline at end of file
+# test-cred  git clone
```

Comment:
Could you please write about git clone in next line

Reply:
ok will update